.github/workflows/workflow.yml

```diff
@@ -49,7 +49,7 @@ jobs:
             cabal: "2.4.0.1"
             expect-fail: false
           - os: ubuntu-latest
-            ghc: "8.12.0" # A version that will never exist.
+            ghc: "9.0.1" # A version that will never exist.
```

Comment:
9.0.1 definitely exists :)

Perhaps 8.12.0 should be changed to something like 8.20.5 to be more clear that it's a fake version?

Reply:
uh, I don't understand how I managed not to parse that comment! :D

Reply:
Totally happens!